src/nls/messages_en.json

```diff
@@ -60,6 +60,13 @@
     "dashboard.taskRun.status.running": "Running",
     "dashboard.taskRun.status.succeeded": "Completed",
     "dashboard.taskRun.status.waiting": "Waiting",
+    "dashboard.triggerBindings.clearNotification": "Clear Notification",
```

Comment:
To help ensure consistency and reduce the effort required for future translation we should probably define a `dashboard.notification.clear` that we re-use instead of re-defining this for each notification location.

Similar with some of the others, `dashboard.tableHeader.name`, `dashboard.tableHeader.dateModified`, etc.

Maybe even the 'empty state' messages, `No {kind} under any namespace`, `No {kind} under namespace {namespace}`, but let's leave these ones for now. We can review all of our empty states in one go separate to this PR.